### PR TITLE
Add a "Swap My Vote" button to swap cards

### LINKED ISSUE #236: Add stronger CTA on swap selection page
When choosing which user to swap with, there's a list of 'cards' displaying user details, polling data etc., but there's no positive call to action.

It would be nice to make it explicit what is going on, i.e. selecting someone to swap votes with. E.g.

- “Swap with Donna P”? somewhere/top left, possibly as a button
- an “Ask to Swap My Vote” button underneath all the swap detail (note: need to think about how this would sound on a screen reader- lots of identical buttons being read out would be bad)

### PR DESCRIPTION
**Pull request by stevebaxter**
This makes it much clearer what to do when choosing swaps.

Fixes #236.

### CODE REVIEW

File: app/views/user/swaps/_swap_profile_inner.html.haml [hunk 1]
```diff
@@ -12,8 +12,9 @@
     - if other_user.email.present?
       %i.fa.fa-envelope{ title: "Email address potentially available" }
 
-  .profile-constituency.subdued
```
Comment (stevebaxter) [inline app/views/user/swaps/_swap_profile_inner.html.haml:15]: Removed unused class

File: app/views/user/swaps/_swap_profile.html.haml [hunk 2]
```diff
@@ -19,3 +15,8 @@
       .recommendations-header
         Tactical Recommendations
       = render :partial => 'recommendations/recommendation', :collection => recommendations
+
+- if defined? link_to_swap and link_to_swap
```
Comment (stevebaxter) [inline app/views/user/swaps/_swap_profile.html.haml:19]: This is the main change

**Comment by stevebaxter** [comment]
image

**Comment by baob** [comment]
Looks great :)

**Review by aspiers — APPROVED** [review]

**Comment by aspiers** [comment]
Rebasing to fix conflicts

**Pull request merged**